flake.nix

```diff
@@ -0,0 +1,51 @@
+{
+  description = "Real-time multi-physics simulation with an emphasis on medical simulation.";
+
+  inputs = {
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
this "nixos-unstable" branch is a rolling release, but since Nix development cycle is the same as sofa, we could use `nixos-24.05` instead, if we don't forget to bump this every 6 months before sofa release